### PR TITLE
docs: improve main readme

### PR DESCRIPTION
**Pull request by noandrea**
add links to various sections of the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,29 +18,46 @@ For more information on the research behind the cosmos cash protocol, please loo
...
-- `make install`
-- `cosmos-cashd -h`
+### Architecture
 
-### How to run the chain
+The Cosmos Cash approach leverage open standards to reach it's goals and to offer an open model compatible with 3rd party projects that uses the open standards. In particular we use 
+- SSI 
+- DID
+- Verifiable Credentials
+- Zero knowledge proof
```
Comment (barriebyron) [inline README.md:28]:
```suggestion
The Cosmos Cash approach leverages open standards to reach its goals and to offer an open model that is compatible with third-party projects that use the open standards. In particular, the Cosmos Cash project uses: 
- Self-sovereign identity (SSI)
- Decentralized identifier (DID)
- Verifiable credentials (VC)
- Zero-knowledge proofs  
```

File: README.md [hunk 2]
```diff
@@ -18,29 +18,46 @@ For more information on the research behind the cosmos cash protocol, please loo
...
+Links to presentations, discussions and interviews are available in the [presentations page](./docs/Explanation/presentations.md).
 
-## Testnet 
+### Getting started 
 
-The Cosmos Cash testnet coordinates are:
+To get start and contribute to the project visit the [technical setup](./TECHNICAL-SETUP.md) page and the [contributing](./CONTRIBUTING.md) page.
+
+
+A Cosmos Cash testnet is available at this coordinates:
```
Comment (barriebyron) [inline README.md:45]:
```suggestion
To get started and contribute to the project, visit the [technical setup](./TECHNICAL-SETUP.md) page and the [contributing](./CONTRIBUTING.md) page.


A Cosmos Cash testnet is available at these coordinates:
```

File: README.md [hunk 3]
```diff
@@ -18,29 +18,46 @@ For more information on the research behind the cosmos cash protocol, please loo
...
 
 - **Chain ID**: `cosmoscash-testnet`
 - **Token Denom**: `cash`
 - **Genesis**: [cosmos-cash.app.beta.starport.cloud/genesis](https://cosmos-cash.app.beta.starport.cloud/genesis?)
 - **RPC URL**:  `https://rpc.cosmos-cash.app.beta.starport.cloud:443`
-- **Explorer URL**: TBD
+
+A Cosmos Cash DID resolver endpoint (compatible with the [universal resolver](https://github.com/decentralized-identity/universal-resolver) driver specifications) is available at:
```
Comment (barriebyron) [inline README.md:52]:
```suggestion
A Cosmos Cash DID resolver endpoint that is compatible with the [universal resolver](https://github.com/decentralized-identity/universal-resolver) driver specifications is available at:
```

File: README.md [hunk 4]
```diff
@@ -18,29 +18,46 @@ For more information on the research behind the cosmos cash protocol, please loo
...
+```
+https://api.cosmos-cash.app.beta.starport.cloud/identifier/
+```
+
+A universal resolver frontend is also available for testing and verification purposes at:
+- https://uniresolver.cosmos-cash.app.beta.starport.cloud
+
+--- 
+
+Have questions or want to get in touch? drop an email at *cosmos-cash@tendermint.com*
```
Comment (barriebyron) [inline README.md:63]:
```suggestion
Do you have questions or want to get in touch? Send us an email at *cosmos-cash@tendermint.com*.
```

**Review by barriebyron — APPROVED** [review]
minor edits, you know me, I can't resist making our language as clear as a bell.

**Comment by noandrea** [comment]
thanks @barriebyron !

**Pull request merged**